shell.nix

```diff
@@ -1,4 +1,10 @@
-with { pkgs = import ./nix {}; };
-pkgs.mkShell
-  { buildInputs = [ pkgs.devTools.niv ];
-  }
+{ sources ? import ./nix/sources.nix
```

Comment:
Off-topic: I have assumed that overridable `sources`s are useful, but I have yet to use it. It'd be hard to use with tools like `stack` anyway. I'm not even sure how I'd use it with `arion`.

Reply:
I left them around just not to change things too much :) But I agree.